packages/expo-stories/README.md

```diff
@@ -0,0 +1,11 @@
+# Expo Stories
```

Comment:
Could be nice to pimp this readme a bit more, e.g. like `packages/html-elements`?